src/managers/NewsManager.js

```diff
@@ -0,0 +1,81 @@
+const ReqUtil = require('../util/ReqUtil');
+const CacheManager = require('./CacheManager');
+const News = require('../structures/News');
+
+class NewsManager{
+    constructor(client){
+        /**
+         * The client instance.
+         * @type {Client}
+         */
+        this.client = client;
+
+        /**
+         * The cache manager
+         * @type {CacheManager} 
+         * @private
+         */
+        this._cache = new CacheManager(client, News);
+    }
+
+    /**
+     * Fetches a Trackmania splashscreen and returns its data.
+     * @param {News} newsId The splashscreen ID
+     * @param {Boolean} cache Whether to get the news from cache or not
+     * @returns {Promise<News>} The splashscreen
+     * @example 
+     * client.news.get(143).then(news => {
+     *     console.log(news.title);
+     * });
+     */
+    async get(newsId, cache = this.client.options.cache.enabled){
+        if (cache && this._cache.has(newsId)) {
+            return this._cache.get(newsId);
+        } else {
+            return await this._fetch(newsId, cache);
+        }
+    }
+        
+    /**
+     * Fetches a splashscreen and returns its data
+     * @param {String} newsId The splashscreen ID
+     * @param {Boolean} cache Whether to cache the news or not
+     * @returns {News} The splashscreen
+     * @private
+     */
+    async _fetch(newsId, cache = this.client.options.cache.enabled){
+        const news = this.client.options.api.paths.tmio.tabs.news;
+        const res = await this.client._apiReq(`${new ReqUtil(this.client).tmioAPIURL}/${news}/0`);
+
+        if (cache) {
+            // Get page_max from response, if set to 1, then there is only one page
+            // else we need to fetch all pages given by page_max
+            // and cache all splashscreens
```

Comment:
Do we really need to cache all splashscreens? If I call `_fetch()` with an ID and `cache=true` then I'd only expect it to cache that one splashscreen.
It's probably not a big deal since there's not many splashscreens at the same time typically - but I feel like this is a bit unintuitive.